lib/tests/streamlit/connections/snowpark_connection_test.py

```diff
@@ -133,8 +133,6 @@ def test_retry_behavior(self):
         MagicMock(),
     )
     def test_retry_fails_fast_for_most_errors(self):
-        from snowflake.snowpark.exceptions import SnowparkServerException
-
```

Comment:
unrelated change to remove an unused import, but I don't think it's worth creating a separate PR for this